JamfSync/Model/JamfProPackageUApi.swift

```diff
@@ -9,14 +9,17 @@ class JamfProPackageUApi: JamfProPackageApi {
         guard let url = jamfProInstance.url else { throw ServerCommunicationError.noJamfProUrl }
 
         var packages: [Package] = []
-        let packagesUrl = url.appendingPathComponent("/api/v1/packages")
+        let pageParameters = [URLQueryItem(name: "page", value: "0"), URLQueryItem(name: "page-size", value: "2000"), URLQueryItem(name: "sort", value: "id%3Aasc")]
```

Comment:
I'm curious what that "id%3Aasc" is all about...can you fill me in? Actually a comment here might be helpful because it isn't self explanatory. Assigning that value to a variable with a descriptive name might be even better. Also, it would be better if we could loop through each page until jsonPackages.totalCount is >= jsonPackages.results.count.

Reply:
sorts the returned packages by id in ascending order.
Added a loop to get all packages.